library/general/src/modules/Popup.rb

```diff
@@ -1913,6 +1910,20 @@ def ShowFile(headline, filename)
       nil
     end
 
+  private
+
+    # Adapt a message to be shown as plaintext in a Richtext widget
+    #
+    # * Escape tags
+    # * Replace "\n" by "<br>"
+    #
+    # @param [String] message Message to be converted
+    # @return [String] Converted message
+    def convert_to_plaintext(message)
+      String.EscapeTags(message).split("\n").join("<br>")
```

Comment:
NTH: The `split` + `join` trick is a relic from the YCP times. Use native Ruby [`gsub`](http://ruby-doc.org/core-2.1.4/String.html#method-i-gsub) (or [`tr`](http://ruby-doc.org/core-2.1.4/String.html#method-i-tr) which is better in this case).